### ISSUE TITLE
Accessibility - Main Screen - Missing Heading

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#20]
Why is there no heading on this page? I would expect the text “Sweat & Toil’ to be a heading so people using TalkBack can it to navigate the page with.

**Comment by aquilent-dgemoets** [comment]
Talk Balk reads the Sweat & Toil heading on our device.  What phone and Android version were used when it was not behaving like a heading?

**Comment by aquilent-dgemoets** [comment]
Unlike iOS VoiceOver, Android TalkBack does not automatically announce/identify headings as such, including screen titles and tab titles; this must be added explicitly by the developer.  We will add heading indicators (in "alt tags") in as many places as time permits-- in some cases this will be easy, in other cases it very much will not be (also applies to #22, #30, #31, #36, #37, #39, #40, #42, and #43).

**Comment by jfc3-dol** [comment]
I reviewed the ILAB 'Sweat and Toil' application on OPA's 'Galaxy Tab S' and an ILAB employees 'LG Tribute 2' and both did not read any headings. 

From my understanding two of the 'More information' screens (About the App and Methodology) where HTML pages and had the headings in them. 

I thought the OSHA 'Heat Index' application had similar screens and the headings with their proper level were read for HTML screens before. The other Android screens that had what equated to headings were read too.

**Comment by aquilent-dgemoets** [comment]
I checked the repository for the (unpublished) Android version of the OSHA 'Heat Index' application and didn't see any added accessibility code/configuration.  Could you confirm that the behavior you are remembering was for the Android version of the app?  If so, maybe we can find out how they did it.

**Comment by aquilent-dgemoets** [comment]
Should this one be closed like the similar screen title heading issues?

**Comment by jfc3-dol** [comment]
I was going to leave it open until the three other issues in it are closed.

**Comment by aquilent-dgemoets** [comment]
I don't know what three other issues you're referring to.  Can you be more specific?

However, in the latest beta (from 12/4), the three selections on the main screen (Countries, Goods, Exploitation Types) are now identified as buttons.

**Comment by nmangu** [comment]
This is verified and validated as working.

**Comment by aquilent-dgemoets** [comment]
Great.  Are you authorized to close issues, or are there other open aspects to this issue?

**Comment by nmangu** [comment]
Yes, we are authorized to close issues that are fixed. However, since these were opened by the Sr. Accessibility Specialist, he might want to review these once he would be back on Tuesday. You may go ahead and consider these (the ones that are fixed, re-tested and passed) as closed. Please be aware that the ones that were beyond the scope of the fix would still be considered as 'open' till our Sr. Accessibility Specialist reviewed and closed them himself.

**Comment by jfc3-dol** [comment]
Methodology screen has links that should be heading level two (h2) items. Currently, they look like headings but don't tell the user they are.

**Comment by carolynhuang** [comment]
Please clarify the issue that needs to be resolved, as it seems this is linked to a few other issues that were resolved.

**Comment by jfc3-dol** [comment]
According to issues #22 and #39 still are having issues with headings so I have left this issue open along with the "Methodollogy" screen has look and should be headings but are not being rad as headings. My guess is the way they are coded to be a link is causing TalkBack to only mention them as links and not as a heading too.

**Comment by jfc3-dol** [comment]
There is still no heading on the home screen but the Methodology screen does have one now.

**Comment by jfc3-dol** [comment]
We used our Samsung Galaxy S7 (SM-G930V) to do this testing with.

**Comment by jfc3-dol** [comment]
TalkBack is not reading any text as a heading on this screen.

**Comment by jfc3-dol** [comment]
TalkBack is still not reading any text as a heading on this screen. I would expect the text "Sweat & Toil" in the banner/header section of the application to be a heading.

So I'm not closing the issue.

**Comment by jfc3-dol** [comment]
@pbhatt17 

TalkBack is still not reading any text as a heading on this screen. I would expect the text "Sweat & Toil" in the banner/header section of the application to be a heading.

So I'm not closing the issue.
